requirements.txt

```diff
@@ -0,0 +1,4 @@
+git+https://github.com/edx/XBlock.git#egg=XBlock==0.4.12
+git+https://github.com/edx/xblock-utils.git#egg=xblock-utils==1.0.2
+django==1.8.12
```

Comment:
Maybe we should be less specific on the `django` requirement version.  
We have some customers on `named-release/dogwood.1` still, which uses `django==1.8.7`